client/components/AddTestToQueueWithConfirmation/index.jsx

```diff
@@ -129,6 +133,37 @@ function AddTestToQueueWithConfirmation({
         );
     };
 
+    const renderErrorDialog = () => {
+        return (
+            <BasicModal
+                show={errorMessage}
+                title={'Conflicting Report Found'}
```

Comment:
No need to use interpolation for static values.

```suggestion
                title='Conflicting Report Found'
```

Reply:
Thanks. Done.